README.md

```diff
@@ -1,8 +1,44 @@
-
-  # bs-benchmarkjs
-
-  Very very basic bindings to [Benchmark.js](https://benchmarkjs.com/)
-
-  ## Usage
-
-  See [examples](https://github.com/glennsl/bs-benchmarkjs/tree/master/examples)
\ No newline at end of file
+
+# bs-benchmarkjs
+
+BuckleScript external type bindings to [Benchmark.js](https://benchmarkjs.com/), an excellent JavaScript performance benchmark library.
+
+## Usage
+

```

Comment:
Very nice. I think it's a good idea to include an installation section as well to make it painfully obvious how to get started with it. I require it for redex, so pretty much every BuckleScript package has one you can refer to as an example.

Reply:
I wholeheartedly agree.

Reply:
I included the instructions for npm/yarn installation. I think I just need the proper permissions to actually publish the package.